[2023-04-14 17:28:22,330] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.upload_data_big_query manual__2023-04-14T17:28:18.317066+00:00 [queued]>
[2023-04-14 17:28:22,336] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.upload_data_big_query manual__2023-04-14T17:28:18.317066+00:00 [queued]>
[2023-04-14 17:28:22,337] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 17:28:22,338] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-04-14 17:28:22,338] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 17:28:22,346] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): upload_data_big_query> on 2023-04-14 17:28:18.317066+00:00
[2023-04-14 17:28:22,353] {standard_task_runner.py:52} INFO - Started process 511 to run task
[2023-04-14 17:28:22,357] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'vic_investigacion_ETL', 'upload_data_big_query', 'manual__2023-04-14T17:28:18.317066+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/vic_investigacion_dag.py', '--cfg-path', '/tmp/tmpv96bbodg', '--error-file', '/tmp/tmpcy1of415']
[2023-04-14 17:28:22,358] {standard_task_runner.py:80} INFO - Job 153: Subtask upload_data_big_query
[2023-04-14 17:28:22,411] {task_command.py:371} INFO - Running <TaskInstance: vic_investigacion_ETL.upload_data_big_query manual__2023-04-14T17:28:18.317066+00:00 [running]> on host 6051ec54e0cc
[2023-04-14 17:28:22,459] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vic_investigacion_ETL
AIRFLOW_CTX_TASK_ID=upload_data_big_query
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T17:28:18.317066+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-14T17:28:18.317066+00:00
[2023-04-14 17:28:22,460] {logging_mixin.py:115} INFO - Uploading Data to big query
[2023-04-14 17:28:22,461] {python.py:173} INFO - Done. Returned value was: None
[2023-04-14 17:28:22,471] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=vic_investigacion_ETL, task_id=upload_data_big_query, execution_date=20230414T172818, start_date=20230414T172822, end_date=20230414T172822
[2023-04-14 17:28:22,490] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-14 17:28:22,514] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
